src/components/search-wizard/QueryLayer.jsx

```diff
@@ -1,9 +1,106 @@
+  "Related Tables": "None",
+  "Document Search": "https://documents.deq.utah.gov/TabsPage.aspx?AI_PageConfigID=100001&DivName=DDW",
+  "GRAMA Request": "https://deq.utah.gov/general/records-request-government-records-access-and-management-act-grama",
+  "Permit Information": "https://documents.deq.utah.gov/TabsPage.aspx?AI_PageConfigID=100123&DivName=DAQ",
+  "Additional Information": "https://documents.deq.utah.gov/TabsPage.aspx?AI_PageConfigID=100003&DivName=DDW",
+  "Comments": "",
+  "Feature Service": "",
+  "Coded Values": ""
+}
+*/
+export default function QueryLayer({
+  config,
+  selected,
+  onSelectedChange,
+  // filter,
+  // onFilterChange,
+}) {
+  const id = `query-layer-${config[fieldNames.queryLayers.uniqueId]}`;
 
-  // todo - use logEvent from 'firebase/analytics' to log which layers are selected
-  return <Checkbox name={id} label={config[fieldNames.queryLayers.name]} />;
+  return (
+    <div className="my-2 flex items-center justify-between">
+      <Checkbox
+        name={id}
+        label={config[fieldNames.queryLayers.name]}
+        onChange={onSelectedChange}
+        checked={selected}
+      />
+      <Tooltip
+        trigger={
+          <a
+            href={config[fieldNames.queryLayers.metadataLink]}
+            target="_blank"
+            rel="noreferrer"
+          >
+            <Icon
+              name={Icon.Names.help}
+              className="text-gray-600"
+              label="more info"
+              size="xs"
+            />
+          </a>
+        }
+      >
+        <div className="max-w-md">
+          {config[fieldNames.queryLayers.layerDescription]}
+        </div>
+      </Tooltip>
+    </div>
+  );
 }
+
+// TODO: this should be some sort of check that is performed before we
+// create the query layer component
+const requiredConfigFields = [
+  fieldNames.queryLayers.uniqueId,
+  fieldNames.queryLayers.name,
+  fieldNames.queryLayers.metadataLink,
+  fieldNames.queryLayers.layerDescription,
+];
+const configShape = Object.values(fieldNames.queryLayers).reduce(
+  (obj, value) => {
+    obj[value] = requiredConfigFields.includes(value)
+      ? PropTypes.string.isRequired
+      : PropTypes.string;
+    return obj;
+  },
+  {}
+);
+QueryLayer.propTypes = {
+  config: PropTypes.shape(configShape),
+  selected: PropTypes.bool.isRequired,
+  onSelectedChange: PropTypes.func.isRequired,
+  // filter: PropTypes.object,
+  // onFilterChange: PropTypes.func.isRequired,
+};
```

Comment:
**thought**: would this be better served as a yup schema that you can validate? I think proptypes are only shown during development.

Reply:
Great idea. I've implemented this with yup.